src/components/dropdown-menu/ResponsiveMenuContainer.scss

```diff
@@ -0,0 +1,60 @@
+@import '../../styles/variables';
+
+.menu-container {
+    position: relative;
+    top: 0;
+    left: 0;
+    z-index: 9999;
```

Comment:
Is this a safe assumption? We've seen issues with competing z-index values in the past.